config.toml

```diff
@@ -11,7 +11,7 @@ themesDir = "node_modules/"
   google_tag_manager = "GTM-5WLCZXC"
   description = "iot.eclipse.org is where you can learn about the technologies developed at Eclipse to make Internet of Things (IoT) development simpler. These technologies aim at establishing an open, end-to-end, IoT stack."
   subtitle = ""
-  seo_title_suffix = " | IoT development made simple - iot.eclipse.org"
```

Comment:
With the changes you made, we probably don't need a suffix but we probably do need one for all the other page. 

I would recommend that we find a solution for keeping a seo suffix.

Reply:
Any progress on this?

Reply:
I created the following PR so that we can also update the prefix on pages throughout our Hugo sites:
https://github.com/EclipseFdn/hugo-solstice-theme/pull/118

Reply:
I think your patch for Hugo is useful but would require us to set a seo_title_suffix on each page of the site. W

We won't do that, I think we need a simplier solution.

Reply:
We could set a default suffix and override it for the home page which is what we want to do for IoT.

Reply:
This is a good example:
https://deploy-preview-386--eclipsefdn-iot.netlify.app/community/resources/case-studies/

The title is set to  "Case Studies" but I would expect something like "Case Studies - Eclipse IoT". This is something I would expect to come from seo_title_suffix in config.toml

Reply:
Using the patch I created for the Hugo theme, we could update the suffix in the config.toml to " - Eclipse IoT". And in the index.md of the front page change it to " - Something else". Wouldn't that work?